src/routes/home/components/AnnouncementDialog.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <BaseDialog
+    id="announcement-dialog"
+    v-model="show"
+    variant="light"
+    :opacity="1"
+    data-cy="announcement-dialog"
+    scrollable
```

Comment:
we should set this to dismissable
```suggestion
    scrollable
    :persistent="false"
```